apps/full-site-editing/full-site-editing-plugin/starter-page-templates/page-template-modal/components/template-selector-item.js

```diff
@@ -49,18 +55,44 @@ const TemplateSelectorItem = props => {
 
 	const labelId = `label-${ id }-${ value }`;
 
+	/**
+	 * Determines (based on arbitary "mobile" breakpoint) whether or not
+	 * the Template selection UI interaction model should be select and confirm
+	 * or simply a single "tap to confirm". The reason for this is that on larger screens
+	 * the large preview is visible which necessitates the double interaction. Without this
+	 * then only a single tap is necessary.
```

Comment:
It would be less arbitrary if we differentiated it by whether we show the large preview or not. Which is what it comes down to, no?

Reply:
Yeh I agree. It's totally dependent on the presence of the larger preview rather than whether it's "mobile" or desktop. We should bring this into our internal conversations for clarity as well.

Updated code and comments to make this ultra clear.